creating_images/sti_testing.adoc

```diff
@@ -143,45 +143,54 @@ a new STI builder image.
 ----
 {
   "kind": "BuildConfig",
-  "apiVersion": "v1beta1",
-  "metadata":{
+  "apiVersion": "v1",
+  "metadata": {
     "name": "ruby-20-centos7-build"
   },
-  "triggers": [
-    {
-      "type": "github",
-      "github": {
-        "secret": "secret101"
+  "spec": {
+    "triggers": [
+      {
+        "type": "github",
+        "github": {
+          "secret": "secret101"
+        }
       }
-    },
-  ],
-  "parameters": {
-    "source" : {
-      "type" : "Git",
-      "git" : {
+    ],
+    "source": {
+      "type": "Git",
+      "git": {
         "uri": "git://github.com/openshift/sti-ruby.git"
       }
     },
     "strategy": {
       "type": "Custom",
       "customStrategy": {
-        "image": "openshift/sti-image-builder",
-        "exposeDockerSocket": true,
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "sti-image-builder:latest"
```

Comment:
this won't work unless you also have them setup the imagestream.

or you can change the Kind to DockerImage and make the name be "openshift/sti-image-builder" again.